.drone.star

```diff
@@ -254,6 +254,7 @@ def gui_tests(ctx, trigger = {}, depends_on = [], filterTags = [], version = "da
                              "SERVER_INI": "/drone/src/test/gui/drone/server.ini",
                              "SQUISH_PARAMETERS": squish_parameters,
                              "STACKTRACE_FILE": STACKTRACE_FILE,
+                             "LC_TIME": "de_DE.UTF-8",
```

Comment:
Setting environment like this is not working for some reason. I will remove it.

Reply:
May be this sets environment for squishrunner only, and not for squishserver!!! 🤔 Otherwise it should have worked